apps/drupal-default/particle_theme/templates/block/block--utc-card.html.twig

```diff
@@ -40,131 +37,142 @@
-				{% set button_style_select = content["#block_content"].field_card_button_type.0.target_id %}
-				{% set button_style = drupal_field('field_css_class', 'taxonomy_term', button_style_select)["#object"].field_css_class.value %}
-				{% set card_link_target = card_link_target.0 == On ? '_blank' %}
-				{% include 'block--utc-card-base.html.twig' with item_card_1 %}
-			</div>
-		{% if item_card_2.card_placeholder %}
-			<div class="col-sm-12 col-lg-4 utc-card-grid__container utc-card-placeholder">
-		{% else %}
-			<div class="col-sm-12 col-lg-4 utc-card-grid__container">
-		{% endif %}
-				{% set button_style_select = content["#block_content"].field_card_button_type_2.0.target_id %}
-				{% set button_style = drupal_field('field_css_class', 'taxonomy_term', button_style_select)["#object"].field_css_class.value %}
-				{% include 'block--utc-card-base.html.twig' with item_card_2 %}
-			</div>
-		{% if item_card_3.card_placeholder %}
-			<div class="col-sm-12 col-lg-4 utc-card-grid__container utc-card-placeholder">
-		{% else %}
-			<div class="col-sm-12 col-lg-4 utc-card-grid__container">
-		{% endif %}
-				{% set button_style_select = content["#block_content"].field_card_button_type_3.0.target_id %}
-				{% set button_style = drupal_field('field_css_class', 'taxonomy_term', button_style_select)["#object"].field_css_class.value %}
-				{% include 'block--utc-card-base.html.twig' with item_card_3 %}
-			</div>
-		</div>
-		{% if item_card_3.card_placeholder and item_card_2.card_placeholder and item_card_1.card_placeholder %}
-		<h2 class="text-center"> Do not hide all the cards </h2>
-		{% endif %}
-	{% endif %}
-
-{% endblock %}
\ No newline at end of file
+    card_button_target: content.field_card_button_target_3[0]|render == 'On' ? '_blank',
+    card_link_target: content.field_card_link_target_3[0]|render == 'On' ? '_blank',
+  } %}
+
+  {% set item_card_4 = {
+    card_placeholder: content.field_card_4_placeholder[0]|render == 'On' ? 'Off',
+    card_link: content.field_card_link_4.0['#url'],
+    card_title: content.field_card_title_4|field_value,
+    card_image: content.field_card_image_4|field_value,
+    card_icon: content.field_card_icon_4|field_value,
+    card_icon_color: content.field_card_icon_color_3|field_value|render,
+    card_text: content.field_card_body_4|field_value,
+    card_button_text: content.field_card_button_4.0['#title'],
+    card_button_url: content.field_card_button_4.0['#url'],
+    card_button_target: content.field_card_button_target_4[0]|render == 'On' ? '_blank',
+    card_link_target: content.field_card_link_target_4[0]|render == 'On' ? '_blank',
+  } %}
+  {% set card_attributes = create_attribute() %}
+  {% if card_count == 2 %}
+    {% set classes = [
+      'utc-card-grid__container',
+      'col-sm-12',
+      card_stack ? 'col-lg-12' : 'col-lg-6',
+    ] %}
+    {% if item_card_2.card_placeholder and item_card_1.card_placeholder %}
+      {% set no_hide = "Do not hide all the cards." %}
```

Comment:
@UTCbridget do we need to keep this do not hide? It looks like maybe there is a script handling this alert?

Reply:
I believe I added that. Only in case the javascript fails and they decide to hide all the cards. 
I'm a little hesitant to say to remove it only because there are pages that are just cards. If they ever make them invisible by mistake someone will double the markup on that page.
It's a fail-safe to the javascript. We could remove it but then there will be no visual indication that something is buried on that node(if the javascript failed or someone a bug lets all cards be hidden). Thoughts @brittrich  @UTCbridget? Is that possibility remote enough?

Reply:
Yes, there is a script handling this. I think this was put as a precaution?

Reply:
Actually on a second thought less leave it there for now. Because if we upgrade classy to classy9 next year some class names might change. Some core classnames, the issue with that is sometimes the javascript is looking for a certain id or class on which case it might not load.

Reply:
Sounds good, I'll leave it in!